CarSharingApp.PublicApi/CarSharingApp.PublicApi.csproj

```diff
@@ -12,6 +12,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Seq.Extensions.Logging" Version="6.1.0" />
```

Comment:
let's configure versions separately using targets

Reply:
Added Directory.Build.targets file and using it as a file for updating versions of the same packages into different projects.